src/app/modules/vault-filter/components/folder-filter.component.html

```diff
@@ -0,0 +1,87 @@
+<ng-container *ngIf="!hide">
+  <div class="filter-heading">
+    <button
+      class="toggle-button"
+      [attr.aria-expanded]="!isCollapsed(foldersGrouping)"
+      appStopProp
+      (click)="toggleCollapse(foldersGrouping)"
+      aria-expanded="!isCollapsed(foldersGrouping)"
+    >
+      <i
+        class="bwi bwi-fw"
+        title="{{ 'toggleCollapse' | i18n }}"
+        aria-hidden="true"
+        [ngClass]="{
+          'bwi-angle-right': isCollapsed(foldersGrouping),
+          'bwi-angle-down': !isCollapsed(foldersGrouping)
+        }"
+      ></i>
+    </button>
+    <h2>&nbsp;{{ foldersGrouping.name | i18n }}</h2>
+    <button
+      class="add-button"
+      appBlurClick
```

Comment:
You didn't use this directive in the `collection-filter.component` - is it needed here?

Reply:
[[bug] Remove unused button directives from vault filters](https://github.com/bitwarden/desktop/pull/1487/commits/d0b1bd00796bedd7f93b341e571355a6990eef6f)